spec/controllers/v1/inventory_invitations_controller_spec.rb

```diff
@@ -0,0 +1,136 @@
+require 'spec_helper'
+
+describe V1::InventoryInvitationsController do
+  render_views
+
+  describe '#create' do
+    context 'without user' do
+      let(:inventory) { FactoryGirl.create(:inventory) }
+
+      before(:each) do
+        sign_out :user
+        post :create, inventory_id: inventory.id, format: :json
+      end
+
+      it 'requires logged in user' do
+        expect(response).to have_http_status(:unauthorized)
+      end
+    end
+
+    context 'as non-facilitator' do
+      let(:user) { FactoryGirl.create(:user) }
+      let(:inventory) { FactoryGirl.create(:inventory) }
+
+      before(:each) do
+        sign_in user
+        post :create, inventory_id: inventory.id, format: :json
+      end
+
+      it 'requires a facilitator to create a invite' do
+        expect(response).to have_http_status(:forbidden)
+      end
+    end
+
+    context 'as facilitator' do
+      let(:inventory) { FactoryGirl.create(:inventory, :with_facilitators) }
+      let(:facilitator_user) { inventory.facilitators.first.user }
+      let(:email) { 'john_doe@gmail.com' }
+
+      context 'inviting user for second time' do
+        let(:existing_invitation) { FactoryGirl.create(:inventory_invitation, inventory: inventory) }
+
+        before(:each) do
+          sign_in facilitator_user
+          post :create,
+            inventory_id: inventory.id,
+            first_name: 'john',
+            last_name: 'doe',
+            email: existing_invitation.email,
+            team_role: 'Finance',
+            role: 'facilitator', format: :json
+        end
+
+        it 'returns error status code' do
+          expect(response).to have_http_status(:unprocessable_entity)
+        end
+
+        it 'returns error message' do
+          expect(json["errors"]["email"]).to include("User has already been invited")
+        end
+      end
+
+      context 'inviting user for first time' do
+
+        let(:created_invitation) { InventoryInvitation.where(email: email).first }
+
+        shared_examples 'successful_invitation' do
```

Comment:
where'd be an  ideal place to put these, inside `support`?
Should I make them load automatically or `require` them explicitely?